lib/resque/worker.rb

```diff
@@ -858,5 +857,11 @@ def very_verbose=(value)
 
       @very_verbose = value
     end
+
+    private
+
+    def log_with_severity(severity, message)
```

Comment:
This should maintain the public interface of `log` and `log!` in case people are using them, but they're no longer used internally by `Resque::Worker`. There is still one caller to `log` in the `work` task of `tasks.rb`.